packages/@storefront/core/src/core/tag/mixins/provide-consume.ts

```diff
@@ -34,10 +34,12 @@ export namespace ProvideConsume {
           this.one(Phase.UNMOUNT, () => {
             aliasTag.off(Phase.UPDATE, resetUpdated);
             aliasTag.off(Phase.UPDATED, softUpdateDependant);
+            this.off(Phase.UPDATE, markUpdated);
```

Comment:
In this context, is `this` different from `aliasTag`? If not, is there a reason why we're not using `aliasTag`?

Reply:
Yes. There are two components in this context: `this`, which is the component that consumes the alias, and `aliasTag`, which is the component that provides the alias. Thinking about it, this line might not be necessary because Riot should be doing this cleanup on unmount anyway.

Reply:
Noted. I have 1x follow question re: how the `aliasTag` is marked for garbage collection.